src/main/java/com/microsoft/azure/functions/annotation/KafkaOutput.java

```diff
@@ -95,37 +144,49 @@
      * Default is plaintext
      *
      * security.protocol in librdkafka
+     * 
+     * @return The SASL password.
      */
     String password() default "";
 
     /**
      * Gets or sets the security protocol used to communicate with brokers
      * default is PLAINTEXT
+     * 
+     * @return The protocol.
      */
     BrokerProtocol protocol() default BrokerProtocol.NOTSET;
 
     /**
      * Path to client's private key (PEM) used for authentication.
      * Default ""
      * ssl.key.location in librdkafka
+     * 
+     * @return The ssl.key.location.
```

Comment:
Can we remove the dots or add dots to ssl.ca.location?

Reply:
Done